sale_order_line_position/models/sale_order_line.py

```diff
@@ -35,7 +35,7 @@ def _add_next_position_on_new_line(self, vals_list):
         ]
         if sale_ids:
             ids = tuple(set(sale_ids))
-            self.flush_model()
+            self.env.flush_all()
```

Comment:
`flush_model` is not deprecated and in fact I fixed the old `flush` call here https://github.com/OCA/sale-reporting/pull/257.

Plus, why flushin ALL the models instead of just the ones that you need? In this case just sale_order_line.

Reply:
Hello @simahawk, indeed, the warning no longer exists. I recall it was there :thinking: 

Anyway, I closed this PR